.github/workflows/codeforafrica-deploy-review-app.yml

```diff
@@ -0,0 +1,135 @@
+
+      - name: Push to dokku
+        uses: dokku/github-action@master
+        with:
+          # create a review app
+          command: review-apps:create
+          git_remote_url: ${{ env.DOKKU_REMOTE_URL }}/codeforafrica-dev
+          # specify a name for the review app
+          review_app_name: codeforafrica-review-${{ github.event.pull_request.number }}
+          ssh_private_key: ${{ secrets.SSH_PRIVATE_KEY }}
+          deploy_docker_image: ${{ env.IMAGE_NAME }}:${{ github.sha }}
+
+      - name: Set Review App Domain Name
+        uses: appleboy/ssh-action@v1.0.0
+        with:
+          host: ui-1.dev.codeforafrica.org
+          key: ${{ secrets.SSH_PRIVATE_KEY }}
+          username: azureuser
+          script: |
+            dokku domains:add codeforafrica-review-${{github.event.pull_request.number}} codeforafrica-review-${{github.event.pull_request.number}}.cms.review.codeforafrica.org --no-restart
+            dokku config:set codeforafrica-review-${{github.event.pull_request.number}} PAYLOAD_PUBLIC_APP_URL=http://codeforafrica-review-${{github.event.pull_request.number}}.cms.review.codeforafrica.org --no-restart
+            dokku config:set codeforafrica-review-${{github.event.pull_request.number}} MONGODB_URL=${{ secrets.CODEFORAFRICA_MONGO_URL }}/${{ secrets.CODEFORAFRICA_MONGO_DB_NAME }}-${{github.event.pull_request.number}} --no-restart
+
+      - name: Update Preview URL
+        uses: nefrob/pr-description@v1.0.0
+        with:
+          content: "[Preview URL](http://codeforafrica-review-${{ github.event.pull_request.number }}.cms.review.codeforafrica.org)"
+          regex: "<!-- preview url start regex match -->.*?<!-- preview url end regex match -->"
+          regexFlags: ims
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+  destroy_review_app:
+    runs-on: ubuntu-latest
+    # only run when a pull request is closed
+    if: github.event_name == 'pull_request' && github.event.action == 'closed'
+    steps:
+      - name: Destroy the review app
+        uses: dokku/github-action@master
+        with:
+          # destroy a review app
+          command: review-apps:destroy
+          git_remote_url: ${{ env.DOKKU_REMOTE_URL }}/codeforafrica-dev
+          # specify a name for the review app
+          review_app_name: codeforafrica-review-${{ github.event.pull_request.number }}
+          ssh_private_key: ${{ secrets.SSH_PRIVATE_KEY }}
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v2
+        id: pnpm-install
+        with:
+          version: 8
+          run_install: false
+
+      - name: Drop cloned DB
+        run: |
+          pnpm install --filter=codeforafrica
+          cd apps/codeforafrica
+          pnpm drop-db "${{ secrets.CODEFORAFRICA_MONGO_URL }}/${{ secrets.CODEFORAFRICA_MONGO_DB_NAME }}-${{github.event.pull_request.number}}"
```

Comment:
Still thinking about this.

Reply:
@kilemensi shouldn't we drop a database? won't the number of databases in a cluster be too many?

Reply:
We should... still thinking about the best way for us to do it.